api/coverage.py

```diff
@@ -70,9 +70,32 @@ def process_batch(self, batch):
         )
 
         results = []
+        leftover_identifiers = set()
+        # We grant coverage if an Edition was created from the operation.
         for edition in imported_editions:
-            self.finalize_edition(edition)
-            results.append(edition.primary_identifier)
+            identifier = edition.primary_identifier
```

Comment:
Previously we called finalize_edition, but that code required edition.license_pool to be set, and it's not set if the DataSource don't match. Now we call finalize_license_pool and operate directly on the LicensePool (which may involve creating or setting a presentation_edition).

However, since it's reasonable to create an Edition but no LicensePool when doing an OPDS import, we count the operation as a success if an Edition was created.